packages/quantic/force-app/main/default/lwc/quanticResultQuickview/e2e/pageObject.ts

```diff
@@ -0,0 +1,39 @@
+import type {Locator, Page} from '@playwright/test';
+
+export class ResultQuickviewObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get button(): Locator {
+    return this.page.locator('c-quantic-result-quickview button');
+  }
+
+  get buttonIcon(): Locator {
+    return this.button.locator('lightning-icon');
+  }
+
+  get buttonTooltip(): Locator {
+    return this.button
+      .locator('..')
```

Comment:
? why the .locator('..') ?

Reply:
J'avais besoin d'aller dans le parent, mais je pourrais faire un autre getter sinon!